src/main/java/com/google/sps/data/Coordinates.java

```diff
@@ -6,15 +6,14 @@
 import com.google.gson.annotations.SerializedName;
 
 /** 
- * This class represents the coordinates of a site location provided by NSW GOV API for each site avaialble for fitering.
+ * This class represents the coordinates of a site location.cd 
```

Comment:
Looks like a cd was typed in the wrong place :)

Reply:
ah!! silly me